packages/iTwinUI-react/src/core/Dialog/DialogTitleBar.tsx

```diff
@@ -48,14 +49,32 @@ export const DialogTitleBar = Object.assign(
       titleText,
       isDismissible = dialogContext.isDismissible,
       onClose = dialogContext.onClose,
+      isDraggable = dialogContext.isDraggable,
       className,
+      onPointerDown: onPointerDownProp,
       ...rest
     } = props;
+
+    const { onPointerDown } = useDialogDragContext();
+    const handlePointerDown = React.useCallback(
+      (event: React.PointerEvent<HTMLDivElement>) => {
+        if (!event.defaultPrevented) {
+          onPointerDown?.(event);
```

Comment:
this needs to be switched with the below one.. otherwise that `defaultPrevented` will do nothing and user has no chance to set it (unless he applies event on child, but still.. wont fully work)

Reply:
Switched